src/main/drivers/sdcard_spi.c

```diff
@@ -603,12 +628,24 @@ static bool sdcard_isReady(void)
  */
 static sdcardOperationStatus_e sdcard_endWriteBlocks(void)
 {
+    uint8_t token = SDCARD_MULTIPLE_BLOCK_WRITE_STOP_TOKEN;
     sdcard.multiWriteBlocksRemain = 0;
 
-    // 8 dummy clocks to guarantee N_WR clocks between the last card response and this token
-    spiBusTransferByte(&sdcard.busdev, 0xFF);
+    // Note that this does not release the CS at the end of the transaction
+    busSegment_t segments[] = {
```

Comment:
Should this be `static const`?

Reply:
No, for the reasons above.

I don't believe the call to spiSequence should be specifying that the `segments` parameter is constant because it would be perfectly well behaved for a callback on one segment to modify the following segment, for example, based on its result.